__tests__/basics.test.ts

```diff
@@ -191,5 +198,45 @@ describe('basics', () => {
         expect(res.message.statusCode).toBe(404);
         let body: string = await res.readBody();
         done();
+    });
+    
```

Comment:
Is it worth adding a test for the `Date` parsing?

Reply:
I'll create an issue for that.  I'll get this in to unblock the other work.